addOns/exim/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Changed
 - Keep the Export menu items sorted alphabetically.
+- Maintenance changes.
```

Comment:
Should mention the change, this affects what the user sees. (Same for other changelogs.)